azext_iot/digitaltwins/providers/deletion_job.py

```diff
@@ -38,9 +38,13 @@ def list(self, top: int = None):  # top is guarded for int() in arg def
         except ErrorResponseException as e:
             handle_service_exception(e)
 
-    def create(self, job_id: Optional[str] = None):
+    def create(self, job_id: Optional[str] = None, timeout_in_min: Optional[int] = None):
+        # Check if error msg is good enough first.
+        # if timeout_in_min < 1:
+        #     raise Exception("Timeout needs to be a postiive integer.")
         job_id = job_id if job_id else DEFAULT_DELETE_JOB_ID_PREFIX + str(uuid4()).replace("-", "")
         self.sdk.config.operation_id = job_id
+        self.sdk.config.timeout_in_minutes = timeout_in_min
```

Comment:
With these options set on `self.sdk.config` - do we need to unset them ever?

Each CLI invocation should "init" the sdk, but curious if this would have any ramifications on subsequent calls made after this one (or if that's even possible)